src/Stratis.Bitcoin/Consensus/Validators/BlockValidator.cs

```diff
@@ -137,7 +137,7 @@ public PartialValidator(IConsensusRuleEngine consensusRules, ILoggerFactory logg
             }
             catch (Exception exception)
             {
-                this.logger.LogCritical("Partial validation callback threw an exception: " + exception.ToString());
+                this.logger.LogCritical("Partial validation callback threw an exception: {0}.", exception);
```

Comment:
we do use ToString() on exceptions

Reply:
Won't the same result be produced?

Reply:
I don't know, but we consistently use it everywhere. I always wanted to check

Reply:
I'm 97% sure it won't affect anything

Reply:
fix it to make it consistent now, we investigate later

Reply:
Fixed